hardware/tests/test_detect.py

```diff
@@ -227,6 +228,114 @@ def test_clean_tuples(self):
                                   b'h\xc3\xa9llo', 1)]),
             [(u'\ufffd' * 4, u'\ufffd' * 4, u'h\xe9llo', 1)])
 
+    @mock.patch.object(detect, 'Popen')
+    @mock.patch('os.environ.copy')
+    def test_detect_auxv_x86_succeed(self, mock_environ_copy, mock_popen):
+        test_data = {
```

Comment:
can you please move all the test_data outside of this file?
like we do for other tests, can be a file in samples

Reply:
Good idea - done